sql/log_event.cc

```diff
@@ -25,7 +25,6 @@
 
 #include "base64.h"            // base64_encode
 #include "binary_log_funcs.h"  // my_timestamp_binary_length
-
 #ifndef MYSQL_CLIENT
```

Comment:
Pls, gives some MTR tests.

Reply:
Time consuming statements in (delete. test) have been added